create-assigned-issue.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+banner = <<-BANNER
+Usage: create-assigned-issue.rb <issue URL>
+  * Copies issue as template
+  * Replaces any references to the assigned user (eg, at-mentions, etc.)
+  * Also "undoes" any checkboxes in issue template (eg, [x] -> [ ])
+  * For each collaborator of repo, creates a new issue, assigned to collaborator
+BANNER
+
+$stderr.sync = true
+require 'octokit'
+require 'optparse'
```

Comment:
Yeah, I kinda wanted to do something like [this](https://gist.github.com/rtomayko/1190547) but it wound up feeling like overkill. But the inspiration was: first explain what this does (but in a way that's re-usable in the script `-h` output), then do some stuff. I dunno. I feel like *everything* wants to be "at the top" but there's only one "top". 

If nothing else, once we pick a "winner" maybe I should update `add-collaborators` to follow the same style. Like, I hear people like consistent style or something?